lib/ex_bitmex/rest/user/wallet_history.ex

```diff
@@ -0,0 +1,39 @@
+defmodule ExBitmex.Rest.User.WalletHistory do
+  alias ExBitmex.Rest
+
+  @type credentials :: ExBitmex.Credentials.t() | nil
+  @type wallet :: ExBitmex.WalletHistory.t()
```

Comment:
How do you feel about changing the name of this type to `wallet_history`?

Reply:
I agree with this change